packages/@aws-cdk/aws-cloudfront/lib/origin.ts

```diff
@@ -172,6 +184,20 @@ export abstract class OriginBase implements IOrigin {
     if (path.endsWith('/')) { path = path.substr(0, path.length - 1); }
     return path;
   }
+
+  /**
+   * Takes origin shield region and converts to CfnDistribution.OriginShieldProperty
+  */
+  private getOriginShield(originShieldRegion:string = ''): CfnDistribution.OriginShieldProperty | undefined {
+    if (originShieldRegion) {
+      if (!(/^\w\w\-[a-z]*\-\d$/).test(originShieldRegion)) throw new Error(`originShieldRegion ${originShieldRegion} is not a valid AWS region.`);
```

Comment:
I'd prefer this validation be moved to the constructor. Also, I could be equally convinced to just remove it; we rarely validate region when input as a parameter, likely to avoid future hassle if/when AWS comes up with a new region that doesn't quite match the format.

Reply:
I agree, I couldn't find many examples of validating for region elsewhere, it's just a thing that bothered me a little. Will remove.